### PR TITLE
[Partners] Adding Partners tile

### PR DESCRIPTION
**Pull request by dcpena**
Adding Partners tile to dev docs

Addresses [PCX-2069](https://jira.cfops.it/browse/PCX-2069)

### CODE REVIEW

File: developers.cloudflare.com/src/components/product-grid.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ $["magic-wan"]            = require("@cloudflare/cloudflare-brand-assets/resourc
 $["network-interconnect"] = require("@cloudflare/cloudflare-brand-assets/resources/product-icons/network-interconnect.js").pathD
 $["pages"]                = require("@cloudflare/cloudflare-brand-assets/resources/product-icons/pages.js").pathD
 $["page-shield"]          = require("@cloudflare/cloudflare-brand-assets/resources/product-icons/page-shield.js").pathD
+$["partners"]             = require("../../../products/partners/src/content/icons/partners.js").pathD
```
Comment (crwaters16) [inline developers.cloudflare.com/src/components/product-grid.js:28]: Guessing this is a temporary home for the icon until we can get it in brand-assets?

File: products/partners/src/content/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Welcome
```
Comment (crwaters16) [inline products/partners/src/content/index.md]: Should be "Overview"

File: products/partners/src/content/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+order: 0
+pcx-content-type: overview
```
Comment (crwaters16) [inline products/partners/src/content/index.md]: Should be "landing-page"

File: products/partners/src/content/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+# Partners
+
+At Cloudflare, we love our partners. Partners play a vital role in Cloudflare's mission to help build a better Internet for customers worldwide.
+
+The Cloudflare Partner Network includes Channel Partners who deliver our security, reliability and performance solutions with a broad range of value-added and managed services, and Alliance Partners who embed Cloudflare into their applications. Our Technology Partners offer customers complementary solutions within the cloud stack.
```
Comment (crwaters16) [inline products/partners/src/content/index.md]: Missing comma after "reliability"

**Review by crwaters16 — APPROVED** [review]
A couple comments:
-The non-landing pages that currently have a pcx-content-type of "overview" should probably use navigation or reference, since they aren't the first page of the tile. 

-Missing a comma after reliability in the partners tile description in the docs-config.js file

File: developers.cloudflare.com/src/components/product-grid.js (hunk 1 above)
Comment (dcpena) [reply developers.cloudflare.com/src/components/product-grid.js:28]: Actually, this is using the new process! We won't need to add it to the brand assets folder anymore :)

File: products/partners/src/content/index.md (hunk 3 above)
Comment (dcpena) [reply products/partners/src/content/index.md]: Fixed

File: products/partners/src/content/index.md (hunk 4 above)
Comment (dcpena) [reply products/partners/src/content/index.md]: Ah, good eye. I copied from the page on the website and didn't catch it. Thanks!

File: products/partners/src/content/index.md (hunk 2 above)
Comment (dcpena) [reply products/partners/src/content/index.md]: Fixed

**Pull request merged**